OpenTabletDriver.Desktop/Interop/Input/Absolute/EvdevVirtualTablet.cs

```diff
@@ -132,6 +127,10 @@ public void SetProximity(bool proximity, uint distance)
         {
             Proximity = proximity;
             Device.Write(EventType.EV_ABS, EventCode.ABS_DISTANCE, (int)distance);
+        }
+
+        public void Finish()
```

Comment:
```suggestion
        public void Finish() => Device.Sync();
```

Cleaner to use => if it's just a one liner

(annoying that you can't do a multi-line suggest, of course remove the curly brackets below)

Reply:
I forgor :skull: Ty and added